code/modules/mob/living/simple_animal/bot/mulebot.dm

```diff
@@ -696,34 +696,37 @@
 	return ..()
 
 /mob/living/simple_animal/bot/mulebot/proc/RunOver(mob/living/carbon/human/H)
-	add_attack_logs(src, H, "Run over (DAMTYPE: [uppertext(BRUTE)])")
-	H.visible_message("<span class='danger'>[src] drives over [H]!</span>", \
-					"<span class='userdanger'>[src] drives over you!</span>")
-	playsound(loc, 'sound/effects/splat.ogg', 50, 1)
+	if(!H.player_logged)//No running over SSD people
```

Comment:
Early return is the preferred method of doing this instead of tabbing the whole thing.

Reply:
Fixed.